test-containers/HelmChartTestWeb.Tests/.env.sample

```diff
@@ -1,8 +1,9 @@
-﻿DB_SERVER_HOST=127.0.0.1
-DB_SERVER_PORT=1533
-DB_CONTAINER_NAME=mssql-server
+﻿DOCKER_ENGINE_HOST=127.0.0.1
+DOCKER_ENGINE_PORT=1234
+DB_SERVER_PORT=1234
+DB_CONTAINER_BASE_NAME=mssql-server
 DB_NAME=aspnet-HelmChartTestWeb
 DB_USERNAME=sa
-DB_PASSWORD=JAsdfi7125o1ih2rt1_@
+DB_PASSWORD=JAsdfi7125o1ih2rt1
```

Comment:
This would violate MS-SQL password policy

Reply:
I reverted the .env.sample file so to serve just for WorkerSettings tests. It;s not gonna be used in mssql server test container tests. The new .env.test-container file is being used in DbServerTests.